R/teal_data-get_code.R

```diff
@@ -0,0 +1,50 @@
+#' Get Code from `teal_data`
+#'
+#' Get code from [`teal_data`]. The extraction of the code limited to specific objects can be performed with the usage
+#' of `names` parameter. If input `teal_data@verified` is `FALSE` the returned code is pre-pended with a warning
+#' stating that the object was not verified.
+#'
+#' @param x [`teal_data`] object.
```

Comment:
No links in argument types, please, that's too much.
```suggestion
#' @param x (`teal_data`) object.
```

Reply:
This comment was about the link.

Reply:
it's no longer a link and the name changed. closing. thanks for participation

Reply:
But you didn't add parentheses and it's now not like other arguments. Throughout `teal`, I might add.

Reply:
totally. my bad. will fix

Reply:
ee57894f32cddb2c85c2c7e7f6c129b49c583989